RF_PBMC.R

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env Rscript
+
+
+# Load libraries
+suppressPackageStartupMessages(library(optparse))
+suppressPackageStartupMessages(library(logging))
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(randomForest))
+suppressPackageStartupMessages(library(Seurat))
+
+ 
+# process_data function follows Seurat tutorial for processing PBMC data set made available by 10x Genomics
+# dataset https://s3-us-west-2.amazonaws.com/10x.files/samples/cell/pbmc3k/pbmc3k_filtered_gene_bc_matrices.tar.gz
+# Seurat tutorial https://satijalab.org/seurat/pbmc3k_tutorial.html
+process_data <- function(input.data){
+	dense.size <- object.size(x = as.matrix(x = input.data))
+	print(dense.size)
```

Comment:
Dense size can be removed you are not using it in the rest of the code. Remove lines 16 and 17. This is a point in the tutorials about how different representations in memory of the data are more or less efficient.

Reply:
removed